### DISCUSSION TITLE
Home Assistant Core (su Docker) vs Home Assistant Core

### DISCUSSION THREAD

**Discussion by gianlucadegiorgio** [indomus/forum#3274] [HUB: Home Assistant]
Consiglio tecnico.
Premessa, uso linux da quando si installava con le immagini sui dischi da 1.44 M, quindi non ho problemi da un punto di vista sistemistico stretto. Conosco pochissimo, invece, il mondo HA.
Quindi da tecnici, quale delle due soluzioni scegliereste? Motivi ?
Grazie

**Comment by KC-inDomus** [comment]
https://indomus.it/guide/home-assistant-hassio-o-home-assistant-core-quale-installazione-fa-al-caso-mio/

**Comment by gianlucadegiorgio** [comment]
Ti ringrazio per la risposta anche se, avendo già letto quell'articolo, lunico punto in cui in qualche modo si toccava l'argomento "installazione su venv" .. era questo :

**N.b. In realtà “Core” può anche essere installato in altre modalità diverse dall’istanziamento Docker, come per esempio quale [applicativo “venv” su Raspberry Pi](https://indomus.it/guide/come-installare-e-configurare-home-assistant-su-un-raspberry-pi-gia-in-uso/). Si tratta però di modalità sempre meno usate per una questione di praticità di gestione e manutenzione, tant’è che su inDomus abbiamo cessato di documentarle.**

In realtà mi sarebbe piaciuto leggere qualcosa di più specifico, magari con qualche esempio argomentativo, che desse forza alla "Nota Bene" riportata nell'articolo.
Comunque .. se nessuno vuole aggiungere qualcosa ... mi farò bastare quello. Grazie a tutti
